packages/wp-now/src/wp-now.ts

```diff
@@ -0,0 +1,168 @@
+    await instance.setup(options);
+    return instance;
+  }
+
+  patchFile = (path, callback) => {
+    this.php.writeFile(path, callback(this.php.readFileAsText(path)));
+  }
+
+  async setup(options: WPNowOptions = {}) {
+    this.options = {
+      ...this.options,
+      ...options,
+    }
+    const { phpVersion, documentRoot, absoluteUrl } = this.options
+    this.php = await PHP.load(phpVersion, {
+      requestHandler: {
+        documentRoot,
+        absoluteUrl,
+        isStaticFilePath: (path) => {
+          try {
+              const fullPath = this.options.documentRoot + path;
+              return this.php.fileExists(fullPath)
+                  && !this.php.isDir(fullPath)
+                  && !seemsLikeAPHPFile(fullPath);
+          } catch (e) {
+              console.error(e);
+              return false;
+          }
+        },
+
+      }
+    })
+    this.php.mkdirTree(documentRoot)
+    this.php.chdir(documentRoot)
+    this.php.writeFile(`${documentRoot}/index.php`, `<?php echo 'Hello wp-now!';`)
+  }
+
+  mountWordpress(fileName = 'latest') {
+    const { documentRoot } = this.options
+    const root = path.join(WORDPRESS_ZIPS_PATH, fileName, 'wordpress')
+    this.php.mount({
+      root,
+    }, documentRoot)
+    this.php.writeFile(
+      `${documentRoot}/wp-config.php`,
+      this.php.readFileAsText(`${documentRoot}/wp-config-sample.php`)
+  );
+  this.patchFile(
+      `${documentRoot}/wp-config.php`,
+      (contents) =>
+          `<?php
+          define('WP_HOME', "${this.options.absoluteUrl}");
+          define('WP_SITEURL', "${this.options.absoluteUrl}");
+          ?>${contents}`
+  );
+  this.php.mkdirTree(`${documentRoot}/wp-content/mu-plugins`);
+  this.php.writeFile(
+      `${documentRoot}/wp-content/mu-plugins/0-allow-wp-org.php`,
+      `<?php
+      // Needed because gethostbyname( 'wordpress.org' ) returns
+      // a private network IP address for some reason.
+      add_filter( 'allowed_redirect_hosts', function( $deprecated = '' ) {
+          return array(
+              'wordpress.org',
+              'api.wordpress.org',
+              'downloads.wordpress.org',
+          );
+      } );`
+  );
+    this.php.cli
+  }
+
+  async runCode(code) {
+    const result = await this.php.run({
+      code,
+    })
+    console.log(result.text)
+    return result
+  }
+
+  async mountSqlite() {
+    const { documentRoot } = this.options
+    const sqlitePluginPath = `${documentRoot}/wp-content/plugins/${SQLITE_FILENAME}`
+    this.php.mkdirTree(sqlitePluginPath)
+    this.php.mount(SQLITE_PATH, sqlitePluginPath)
+    this.php.writeFile(
+      `${documentRoot}/wp-content/db.php`,
+      this.php.readFileAsText(`${sqlitePluginPath}/db.copy`)
+        .replace(/\{SQLITE_IMPLEMENTATION_FOLDER_PATH\}/g, `${documentRoot}/wp-content/plugins/${SQLITE_FILENAME}`)
+        .replace(/\{SQLITE_PLUGIN\}/g, SQLITE_FILENAME)
+    )
+  }
+
+  mount() {
+    this.mountWordpress()
+    this.mountSqlite()
```

Comment:
`mountSqlite` is `async` but this function isn't. we might want to make this `async` and `await this.mountSqlite()` or acknowledge that `mount()` might return before `mountSqlite` has completed (or technically, before it has even begun to process)

Reply:
Good catch!
`mountSqlite` shouldn't be async 👍 
Fixed here: https://github.com/WordPress/wordpress-playground/pull/188/commits/684fb86916de4c27e78c726e9f6584268ce6571d